README.md

```diff
@@ -83,6 +83,29 @@ $ docker run --rm -it -p 9090:9090 -v $PWD/examples/prometheus.yml:/etc/promethe
 ```
 Then head over to http://localhost:9090/graph?g0.range_input=1h&g0.expr=example_value_active&g0.tab=1 or http://localhost:9090/targets to check the scraped metrics or the targets.
 
+# Exposing metrics through HTTPS
+
+```
+$ cat test/tls/web-config.yml
+# Minimal TLS configuration example. Additionally, a certificate and a key file
+# are needed.
+tls_server_config:
+  cert_file: test/tls/server.crt
+  key_file: test/tls/server.key
+
+$ ./json_exporter --config.file examples/config.yml --web.config=test/tls/web-config.yml &
```

Comment:
```suggestion
```

Running:
```
$ ./json_exporter --config.file=examples/config.yml --web.config=web-config.yml &
```

Reply:
Done

Reply:
Seems like you forgot to `s/--config.file ex/--config.file=ex/`